Packs/GoogleBigQuery/Integrations/GoogleBigQuery/GoogleBigQuery_test.py

```diff
@@ -15,3 +18,196 @@ def test_convert_to_string_if_datetime():
     convert_to_string_if_datetime(today)
     test_conversion_for_empty_string = convert_to_string_if_datetime("")
     assert test_conversion_for_empty_string == ""
+
+
+def test_remove_outdated_incident_ids_keep_equal():
+    """
```

Comment:
please update the case docs. see other tests you added also.

Reply:
done